tfx/orchestration/experimental/kubernetes/driver_container_entrypoint.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I would suggest that you split this change into 2 PRs: (1) that provides the base Kubernetes job execution functionality and (2) another where the orchestrator itself runs as a Kubernetes job.

Reply:
Done. This PR is converted to the first PR, and I will submit a separate one in addition.